python/app/dialog.py

```diff
@@ -235,6 +235,9 @@ def __init__(self, parent=None):
         # The current visible tabs. This will change based on the current entity type
         self._current_entity_tabs = []
         self.ui.entity_tab_widget.currentChanged.connect(self._load_entity_tab_data)
+        self.ui.entity_tab_widget.currentChanged.connect(
+            self._update_preset_filters_on_tab_change
+        )
```

Comment:
We can remove this -- the preset filters do not need to be updated on tab change because a filter menu is created for each tab, and so each filter menu for its respective tab will keep track of the preset filters for the tab

Reply:
So initially I had it so that they weren't populated when the tab changed, but this lead to issues. For example if you wanted more dynamic filters which considered the current time or the current sg_location, the filters wouldn't get updated again. If you can think of a better way I'm happy to try something else.

Reply:
I'm good to keep it as is